lang/cs/Org.Apache.REEF.IO/PartitionedData/FileSystem/FileSystemInputPartition.cs

```diff
@@ -123,23 +131,23 @@ private ISet<string> Download()
         /// provided by the Serializer
         /// </summary>
         /// <returns></returns>
-        public T GetPartitionHandle()
+        public IEnumerable<T> GetPartitionHandle()
```

Comment:
I remember for some reason, we changed from returning IEnumerable<T> to T when I wrote the original code. @dkm2110 Do you remember what was that requirement from IMRU?